lib/jsonapi/query_builder/paginator/pagy_countless.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require "pagy"
+require "pagy/extras/items"
+require "pagy/extras/countless"
+
+module Jsonapi
+  module QueryBuilder
+    module Paginator
+      class PagyCountless < BasePaginator
+        include ::Pagy::Backend
+
+        def paginate(page_params)
+          @params = {page: page_params}
+
+          pagination_details, records = pagy_countless collection, page: page_params[:number],
+                                                                   items: page_params[:size],
+                                                                   outset: page_params[:offset]
+          [records, pagination_details]
```

Comment:
```suggestion
          @params = { page: page_params }

          pagination_details, records = pagy_countless(collection, page: page_params[:number],
                                                                   items: page_params[:size],
                                                                   outset: page_params[:offset]
          [records, pagination_details]
```

spaces inside hash are missing, convention to keep the parenthesis if passing params to a method is violated 👮‍♂️

Reply:
Actually, the build failed when I added spaces. 

https://github.com/infinum/jsonapi-query_builder/runs/7514185210?check_suite_focus=true#step:4:9

Reply:
Revert then 👍 didn't know we use the standardrb conventions